packages/core/test-bundles/index.js

```diff
@@ -1,11 +1,11 @@
 import '@cds/core/badge/register.js';
 import '@cds/core/icon/register.js';
 import { ClarityIcons, userIcon } from '@cds/core/icon';
-import { CommonStringsService } from '@cds/core';
+import { I18nService } from '@cds/core/internal';
```

Comment:
This import is a public api so it wouldn't need to be imported from internal. The deep import might be what caused the base bundle size to go up

`import { I18nService } from '@cds/core';`